content/workshops/cascading-layouts.md

```diff
@@ -67,11 +86,16 @@ summary: |
 {% import 'utility.macros.njk' as utility %}
 {% import 'contact.macros.njk' as contact %}
 {% import 'embed.macros.njk' as embed %}
+{% import "quotes.macros.njk" as quotes %}
 
 <div id="workshop-video">
   {{ embed.figure([{ youtube: '8k6m87woEpQ' }]) }}
 </div>
 
+<div id="workshop-video-2">
+  {{ embed.figure([{ youtube: 'De6QMQA2aio' }]) }}
+</div>
+
```

Comment:
Is there a reason to do these separate/stacked, instead of our usual grid-gallery pattern that we use for other talks with multiple videos?